practice/00-HTML-CSS-basics/ExerciseOne.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">	
+<header>
```

Comment:
You should add the header inside the body tag. The head tag is only for metadata information.